apps/judicial-system/web/src/routes/CourtOfAppeal/Result/Result.tsx

```diff
@@ -18,15 +18,13 @@ import Conclusion from '@island.is/judicial-system-web/src/components/Conclusion
 import CaseFilesOverview from '../components/CaseFilesOverview/CaseFilesOverview'
 import CourtOfAppealCaseOverviewHeader from '../components/CaseOverviewHeader/CaseOverviewHeader'
 
-import { courtOfAppealResult as strings } from './Result.strings'
-import { courtOfAppealRuling as rulingStrings } from '../Ruling/Ruling.strings'
-
 import { useIntl } from 'react-intl'
-import { capitalize, formatDate } from '@island.is/judicial-system/formatters'
-import { CaseAppealRulingDecision } from '@island.is/judicial-system/types'
+import { capitalize } from '@island.is/judicial-system/formatters'
 import { titleForCase } from '../../Shared/SignedVerdictOverview/SignedVerdictOverview'
```

Comment:
Please consider moving `titleForCsse` to a shared location, for example to `formHelper` to avoid importing from a different component.